requirements-test.txt

```diff
@@ -445,3 +449,8 @@ zipp==3.21.0 \
     --hash=sha256:2c9958f6430a2040341a52eb608ed6dd93ef4392e02ffe219417c1b28b5dd1f4 \
     --hash=sha256:ac1bbe05fd2991f160ebce24ffbac5f6d11d83dc90891255885223d42b3cd931
     # via importlib-metadata
+
+# WARNING: The following packages were not pinned, but pip requires them to be
+# pinned when the requirements file includes hashes and the requirement is not
+# satisfied by a package already installed. Consider using the --allow-unsafe flag.
+# setuptools
```

Comment:
I believe this is causing the issue in the CI... We might need to add the `--allow-unsafe` parameter to the `pip-compile` so the `setuptools` is included and the installation will not fail...

Reply:
I had to do that on IIB as well: https://github.com/release-engineering/iib/blob/master/tox.ini#L141

Reply:
ty